src/MichelfMarkdown.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Twig\Extra\Markdown;
 
+use League\CommonMark\CommonMarkConverter;
```

Comment:
Actually, the issue is the constructor which should have used MarkdownExtra instead. I've fixed on `twigphp/twig` as this is a read-only repo. Thank you.

Reply:
:)